src/simpleNbox.cpp

```diff
@@ -436,6 +441,28 @@ unitval SimpleNbox::getData(const std::string& varName,
     } else if( varName == D_RF_T_ALBEDO ) {
         H_ASSERT( date != Core::undefinedIndex(), "Date required for albedo forcing" );
         returnval = Ftalbedo.get( date );
+
+        // Partitioning parameters 
+    } else if(varName == D_F_NPPV) {
+        // For the time being, we are only supporting global beta, not biome-specific
```

Comment:
I believe this comment, and the ones like it below, are left over from copying this code block.

Reply:
Good catch. Addressed in 245d998.